tests/test_patch.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-
+import hashlib
 import os
 import time
 import unittest
```

Comment:
In the old test, something weird is up with line 1453, I get a DeprecationWarning on it:

```
========================================== test session starts ===========================================
platform linux -- Python 3.10.9, pytest-7.1.3, pluggy-1.0.0
rootdir: /home/cscorley/git/whatthepatch
collected 52 items

tests/test_apply.py .............                                                                  [ 25%]
tests/test_patch.py .......................................                                        [100%]

============================================ warnings summary ============================================
tests/test_patch.py:1453
  /home/cscorley/git/whatthepatch/tests/test_patch.py:1453: DeprecationWarning: invalid escape sequence '\ '
    text = """---

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
===================================== 52 passed, 1 warning in 3.09s ======================================
```

It happens on main, too, so it's fine; just FYI for next time you're around this :)

Reply:
Interesting -- I can no longer trigger this after merging.

Reply:
Curiosity got the best of me, I figured it out.  Just need to put these into separate files as in PR #58